cmd/test.go

```diff
@@ -116,6 +128,11 @@ func opaTest(args []string) int {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	if testParams.outputFormat.String() == benchmarkGoBenchOutput && !testParams.benchmark {
+		fmt.Fprintf(os.Stderr, "Cannot use output format %s without running benchmarks (--bench)\n", benchmarkGoBenchOutput)
```

Comment:
Nit: Title case error message feels a bit inconsistent with other commands.  Could be wrong.  Do we have similar error messages in other commands?

Reply:
Nah, thats a good catch.